BREAKING.md

```diff
@@ -12,6 +12,7 @@ There are a few steps you can take to write a good change note and avoid needing
 
 ## 0.52 Breaking changes
 - [chaincodePackage removed from Container](#chaincodePackage-removed-from-Container)
+- [close() removed from IDocumentDeltaConnection](#close-removed-from-IDocumentDeltaConnection)
```

Comment:
you lost the actual section in the merge

Reply:
forgot to add this in the last commit 😅